data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/exception/LogstashMappingException.java

```diff
@@ -0,0 +1,13 @@
+package org.opensearch.dataprepper.logstash.exception;
+
+/**
+ * Exception thrown when attempting to map between Logstash configuration model and Data Prepper model
+ *
+ * @since 1.2
+ */
+public class LogstashMappingException extends LogstashConfigurationException{
+
+    public LogstashMappingException(String errorMessage) {
```

Comment:
Per my comment on wrapping `IOException`, you will need another constructor with `(String message, Exception inner)`.

Reply:
Thanks, I'll update LogstashMappingException